Gemfile

```diff
@@ -6,12 +6,14 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 ruby '2.3.1'
 
 gem 'colorize'
+gem 'combine_popolo_memberships', git: 'https://github.com/everypolitician/combine_popolo_memberships.git'
```

Comment:
You're changing a Gemfile without including the Gemfile checklist… (which would in turn make it clearer what mistake you're making here…)